src/assets/icon.tsx

```diff
@@ -0,0 +1,18 @@
+export { ReactComponent as ArrowDown } from './arrow_down.svg';
+export { ReactComponent as ArrowRight } from './arrow_right.svg';
+export { ReactComponent as ArrowUp } from './arrow_up.svg';
+export { ReactComponent as Clap } from './clap.svg';
+export { ReactComponent as Create } from './create.svg';
+export { ReactComponent as Delete } from './delete.svg';
+export { ReactComponent as Bubble } from './emoji_bubble.svg';
+export { ReactComponent as Carrer } from './emoji_career.svg';
+export { ReactComponent as Computer } from './emoji_computer.svg';
+export { ReactComponent as HandsUp } from './emoji_hands up.svg';
```

Comment:
```suggestion
export { ReactComponent as HandsUp } from './emoji_hands_up.svg';
```

띄워쓰기 있어도 괜찮나요? 
사실 실행에는 상관 없긴 한데 그냥 제 눈에 보여버렸습니다 ㅋㅋ

Reply:
오 띄어쓰기가 있었군요! 수정하겠습니다
굿굿굿